### PR TITLE
New reports with full data of users, posts, transfers

### PR DESCRIPTION
**Pull request by sseerrggii**
Permite crear nuevos reports de usuarios, posts y transferencias con toda la información relevante de la base de datos para administradores.

Posts y Transfers deben ir relacionados con member uid para poder hacer las relaciones.

### CODE REVIEW

File: app/decorators/member_complete_report_decorator.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    end
+  end
+
+  def nil_to_empty(field)
+    if field
+      field.to_formatted_s(:db)
+    else
+      ''
+    end
+  end
```
Comment (enricostano) [inline app/decorators/member_complete_report_decorator.rb]:
```ruby
def nil_to_empty(field)
  return '' unless field
  field.to_formatted_s(:db)
end
```

File: app/views/application/menus/_organization_listings_menu.html.erb [hunk 2]
```diff
@@ -6,12 +6,6 @@
   </a>
   <ul class="dropdown-menu" role="menu">
     <li>
-      <%= link_to alpha_grouped_index_tags_path(post_type: "offer") do %>
-        <%= glyph :tags %>
-        <%= t "application.navbar.tags" %>
-      <% end %>
-    </li>
-    <li>
```
Comment (enricostano) [inline app/views/application/menus/_organization_listings_menu.html.erb]: ?

File: app/views/reports/user_complete_list.erb [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<div class="row">
+  <table class="table table-condensed table-bordered table-striped">
+    <thead>
+      <tr>
+        <!-- %th= User.human_attribute_name(:registration_number)
+        -->
```
Comment (enricostano) [inline app/views/reports/user_complete_list.erb]: ?

**Comment by sseerrggii** [comment]
@sauloperez I need help with all transfer report

**Comment by sauloperez** [comment]
Ok, I'll take this one over.

**Comment by sseerrggii** [comment]
Almost a year later ... I finished! Code review please! 

Be kindly please :heart:

File: app/controllers/reports_controller.rb [hunk 4]
```diff
@@ -21,6 +21,24 @@ def user_list
     end
   end
 
+  def user_complete_list
+    @members = current_organization.members.active.
+               includes(:user).
+               order("members.member_uid")
```
Comment (sauloperez) [inline app/controllers/reports_controller.rb]: I would stick to either multiline or single line but not both styles.

File: app/controllers/reports_controller.rb [hunk 5]
```diff
@@ -44,4 +62,44 @@ def post_list
...
     end
   end
+
+  def post_complete_list
+    @posts = current_organization.posts.
+             of_active_members.
+             active.
+             includes(:user, :category).
+             to_a.
+             sort_by { "posts.id" }
```
Comment (sauloperez) [inline app/controllers/reports_controller.rb]: here as well

File: app/controllers/reports_controller.rb [hunk 6]
```diff
@@ -44,4 +62,44 @@ def post_list
...
+        send_data report.run, filename: report.name, type: report.mime_type
+      end
+    end
+  end
+
+  def transfer_complete_list
+    @transfers = current_organization.all_transfers.
+                 includes(movements: {account: :accountable}).
+                 order("transfers.created_at DESC").
+                 uniq
```
Comment (sauloperez) [inline app/controllers/reports_controller.rb]: and here

File: app/views/reports/user_complete_list.erb (hunk 3 above)
Comment (sauloperez) [reply app/views/reports/user_complete_list.erb]: :fire:

**Comment by enricostano** [comment]
@sseerrggii :clap: :clap: :clap: 

You "only" need to add tests...

**Pull request closed without merging**